front/src/modules/rollingStock/components/RollingStockSelector/SearchRollingStock.tsx

```diff
@@ -201,7 +46,7 @@ const SearchRollingStock = ({
       <div className="col-md-5 ml-2 mb-3 d-flex align-items-center flex-wrap">
         <div className="mr-3">
           <CheckboxRadioSNCF
-            onChange={toggleFilter}
+            onChange={(e) => toggleFilter(e.target.name as 'elec')}
```

Comment:
Can you change the other functions onChange ?
```suggestion
            onChange={() => toggleFilter('elec')}
```